lib/checks/keyboard/tabindex.js

```diff
@@ -1 +1 @@
-return node.tabIndex <= 0;
+return node.getAttribute('tabindex') <= 0;
```

Comment:
given we are not reading [tabIndex](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/tabIndex) which would return an integer, and instead an attribute it would make sense to `parseInt` this & handle `NaN` given someone can set `tabindex='something'` that is not a number